types/index.d.ts

```diff
@@ -0,0 +1,285 @@
+import { Account, Provider, Eth, Tx, TransactionReceipt, Transaction, PromiEvent } from 'web3/types';
+import Web3 from 'web3';
+
+// These types are declared to help you keep track of similar data
+export type UserMaticAddress = string;
```

Comment:
in my understanding, `UserMaticAddress` is `tokenContractAddress` on Matic
so maybe variable name can be something else

Reply:
`UserEthAddress` and `UserMaticAddress` are same so this can be `UserAddress`
@ptsayli @shahbhavir

Reply:
what is `EthContractAddress`?